.github/workflows/CI.yml

```diff
@@ -16,13 +16,14 @@ jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
+      max-parallel: 6
```

Comment:
Since UATs for publish command are failing because two or more tests end up calculating the next version of the component at the same and trying to create exactly the same version (out of which only one would succeed). 

Better way to solve this would be to create `com.python.HelloWorld.somerandomstring`(never exists and fallbacks to use 1.0.0) instead of just `com.python.HelloWorld` to prevent test runs from conflicting and being flakey. Though this solves the problem, we need to still cover the case where we want to calculate the next version of an existing component.

Reply:
I agree, this was just placed as a stop-gap solution while I evaluate all publish UAT tests. Until that evaluation and solution, I didn't want workflows to fail.

Reply:
`max-parallel` config has been removed. As now tests are generating unique component names to avoid name & version conflicts.